src/components/CommonComponents/Pagination/index.module.scss

```diff
@@ -0,0 +1,62 @@
+.pagination {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+
+  .pagination-button {
```

Comment:
Nice! Using all our existing tokens is definitely cool!

Reply:
Thanks